.github/workflows/integrationTest.yml

```diff
@@ -452,6 +455,7 @@ jobs:
             terraform init
             if terraform apply --auto-approve \
               -var="ssh_key_value=${PRIVATE_KEY}" -var="github_test_repo=${{env.CWA_GITHUB_TEST_REPO_URL}}" \
+              -var="github_test_repo_branch=${{env.CWA_GITHUB_TEST_REPO_BRANCH}}" \
```

Comment:
This is a duplicate line. I see it 2 lines down.

Reply:
Nice catch. Fixed it !!!